daemon/daemon.go

```diff
@@ -101,6 +101,24 @@ func NewDaemon(cfg *config.Config) *Daemon {
 		return nil
 	}
 
+	// get containerd version to make sure containerd has started
+	_, err = ctrdClient.Version(context.Background())
+	if err != nil {
+		logrus.Errorf("failed to get containerd version: %v", err)
+		return nil
```

Comment:
Do we need to return err to block daemon startup? @wangforthinker 
Just double check.